docs/documentation/quartz-3.x/how-tos/multiple-triggers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Multiple Triggers
+---
+
+One nice thing about Quartz is that you can split the job registration from the 
+triggers. One nice use case is being able to split out the JobData for each
+trigger. 
+
+Below, we have two triggers, each with their own set of data, but
+we only had to register the one job.
+
+
+```csharp
+public Task DoSomething(IScheduler schedule, CancellationToken ct)
+{
+    var job = JobBuilder.Create<AnExampleJob>()
+                        .WithIdentity("customer-process", "group")
+                        .Build();
+    
+    var replace = true;
```

Comment:
I don't mind the style, other alternative I like in code reviews especially is supplying as named parameter like `await schedule.AddJob(job, replace: true, durable: true, ct);`